docs/conf/prague/2020/news/cfp-open.rst

```diff
@@ -0,0 +1,46 @@
+:template: {{year}}/generic.html
+
+.. post:: June 22, 2020
+   :tags: prague-2020, website, cfp, tickets
+
+Announcing Prague Call for Proposals
+====================================
+
+Today we are announcing our `Call for Proposals <https://www.writethedocs.org/conf/prague/2019/cfp/>`_.
+
+Due to the circumstances that you're all familiar with, we're opening it later than usual, and for a shorter length of time.
+And we know that it hasn't been an easy year. Nevertheless we'd love to see you on our online stage in October, to share your ideas with our inspiring community of documentarians.
+
+If there is something you’d really like to see a talk on this year, submit a proposal on it, or refer someone else who would be good!
+
+Call for Proposals
+------------------
+
+Every year, Write the Docs invites people from all across our community to come up on stage to share their insights and experience.
+We're looking for ideas and use-cases from all disciplines and roles, so whether you're a technical writer, editor, developer, UX designer, community manager, or support professional who cares about content and communication, we want to hear from you!
+
+You can read our full `Call for Proposals <https://www.writethedocs.org/conf/prague/{{year}}/cfp/>`__ on the website.
+The Call for Proposals will be open until **Midnight {{tz}} on {{cfp.ends}}**.
+
+Tickets
+-------
+
+In the meantime, you can grab a ticket on our `Tickets <https://www.writethedocs.org/conf/prague/{{year}}/tickets/>`_ page.
+As with the previous years, we have tiered pricing to make it fair for everyone. Please buy the ticket that best fits your situation.
+
+As always, we keep ticket prices low so that it is accessible to the widest range of people possible. However, if you can't afford these prices and still wish to attend, please email us at prague@writethedocs.org, and we can see how we can help.
+
+We will also open a call for volunteers and organizers later, both of which are great opportunities to get involved with the community and also receive a free ticket.
+
+Stay Updated
+------------
+
+Want to find out what's happening with the conference, or enjoy our monthly global community newsletter?
+Sign up to one or more of our `mailing lists <http://eepurl.com/cdWqc5>`_. Your information will never be shared with any third-parties, and you can unsubscribe at any time.
+
+Want to connect with other documentarians in real-time? Join our `Slack <http://slack.writethedocs.org/>`_.
+
+Conclusion
```

Comment:
This feels a bit weak -- I's probably remove the heading and just close it out in the main body text.